File: Daily Test/C++/Find Operator - Maximum Result 7.7.2025.cpp
#include <iostream>
using namespace std;
int main() {
    int x, y, maxVal, res;
    char op = '*';
    cin >> x >> y;
    maxVal = x * y;

    res = x - y;
    if(res > maxVal) { maxVal = res; op = '-'; }

    res = x + y;
    if(res > maxVal) { maxVal = res; op = '+'; }

    if(y != 0) {
        res = x / y;
        if(res > maxVal) { maxVal = res; op = '/'; }
    }

    cout << op << endl;
    return 0;
}

File: Daily Test/Java/Find Operator - Maximum Result 7.7.2025.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int x = sc.nextInt(), y = sc.nextInt();
        int maxVal = x * y;
        char op = '*';

        int res = x - y;
        if(res > maxVal) { maxVal = res; op = '-'; }

        res = x + y;
        if(res > maxVal) { maxVal = res; op = '+'; }

        if(y != 0) {
            res = x / y;
            if(res > maxVal) { maxVal = res; op = '/'; }
        }

        System.out.println(op);
    }
}